src/main/java/org/opentripplanner/graph_builder/module/PruneFloatingIslands.java

```diff
@@ -59,14 +56,12 @@
     public void buildGraph(Graph graph, HashMap<Class<?>, Object> extra) {
         LOG.info("Pruning isolated islands in street network");
         
-        StreetUtils.pruneFloatingIslands(graph, pruningThresholdIslandWithoutStops, 
-        		pruningThresholdIslandWithStops, islandLogFile);
-        if (transitToStreetNetwork == null) {
-            LOG.debug("TransitToStreetNetworkGraphBuilder was not provided to PruneFloatingIslands. Not attempting to reconnect stops.");
-        } else {
-            //reconnect stops on small islands (that removed)
-            transitToStreetNetwork.buildGraph(graph,extra);
```

Comment:
Since the `transitToStreetNetwork` variable was always null, it was safe to remove this code.

Reply:
I never really understood this whole concept of `pruningThresholdIslandWithoutStops` and ` `reconnect stops on small islands`. We probably should not have merged that change without a lot more clarification from the authors. This behavior should be removed, in fact there's already an issue for that at #2639.